plutus-core-interpreter/src/Language/PlutusCore/Interpreter/CekMachine.hs

```diff
@@ -9,71 +9,108 @@
 -- The CEK machine generates booleans along the way which might contain globally non-unique 'Unique's.
 -- This is not a problem as the CEK machines handles name capture by design.
 
+{-# LANGUAGE ExistentialQuantification #-}
```

Comment:
What is this used for?

Reply:
`DynamicBuiltinNameMeaning` is existential, but apparently the pragma is obsolete. Good catch.

Reply:
Fixed.